client/src/Movies/Movie.js

```diff
@@ -33,25 +33,17 @@ const Movie = (props) => {
   const { title, director, metascore, stars } = movie;
   return (
     <div className="save-wrapper">
-      <div className="movie-card">
-        <h2>{title}</h2>
-        <div className="movie-director">
-          Director: <em>{director}</em>
-        </div>
-        <div className="movie-metascore">
-          Metascore: <strong>{metascore}</strong>
-        </div>
-        <h3>Actors</h3>
-
-        {stars.map(star => (
-          <div key={star} className="movie-star">
-            {star}
-          </div>
-        ))}
-      </div>
-      <div className="save-button">Save</div>
+      <MovieCard
+        title={title}
+        director={director}
+        metascore={metascore}
+        stars={stars}
+      />
+      <button className="save-button" onClick={() => saveMovie()}>
```

Comment:
I think you could have done `onClick={saveMovie}`.  That should output the same thing as what you currently have. What happens is the onClick invokes the CB.  Your CB returns a function which is self invoked so it's the same thing.

Reply:
Your right :D